src/screens/ChatScreen/SignedChatScreen.tsx

```diff
@@ -110,8 +106,10 @@ const SignedChatScreen = () => {
       <FlatList
         contentContainerStyle={styles.contentContainerStyle}
         style={styles.chatContainer}
-        data={updatedChats}
+        data={chats}
         inverted={true}
+        windowSize={10}
+        maxToRenderPerBatch={5}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `data` prop of `FlatList` has been changed from `updatedChats` to `chats`. If `updatedChats` was used to maintain chat continuity or perform some other important function, this change could lead to issues. Also, new props `windowSize` and `maxToRenderPerBatch` have been added to the `FlatList`. These props can help optimize rendering, but they should be carefully tested to ensure they don't introduce performance issues or unexpected behavior.

```diff
- data={updatedChats}
+ data={chats}
+ windowSize={10}
+ maxToRenderPerBatch={5}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->